core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/keymanager/imports/ImportKeystoreData.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core.service.http.handlers.keymanager.imports;
+
+import java.util.Objects;
+
+import org.apache.tuweni.bytes.Bytes;
+
+/** Keep the index of the keystore import json and password */
+public class ImportKeystoreData implements Comparable<ImportKeystoreData> {
```

Comment:
Could this be a record instead of a class?

Reply:
We might be able to convert this one to record as we need to modify the "ImportKeystoreResult" state to ERROR which is a member of this class.